docarray/array/mixins/evaluation.py

```diff
@@ -25,13 +30,17 @@ def func(*args, **kwargs):
                     'soon.',
                     DeprecationWarning,
                 )
-        if 'other' in kwargs:
-            kwargs['ground_truth'] = kwargs['other']
-            warnings.warn(
-                '`other` is renamed to `groundtruth` in `evaluate()`, the usage of `other` is '
-                'deprecated and will be removed soon.',
-                DeprecationWarning,
-            )
+        for old_key, new_key in zip(
+            ['other', 'metric', 'metric_name'],
+            ['ground_truth', 'metrics', 'metric_names'],
+        ):
+            if old_key in kwargs:
+                kwargs[new_key] = kwargs[old_key]
+                warnings.warn(
+                    '`other` is renamed to `groundtruth` in `evaluate()`, the usage of `other` is '
```

Comment:
I guess you want to use new key and old key in ther error message

Reply:
oh good catch!